litmus-portal/backend/tools/self-deployer/build/Dockerfile

```
@@ -0,0 +1,28 @@
+#Build Stage
+FROM golang:1.14 AS builder
+
+LABEL maintainer="LitmusChaos"
+
+ADD . /self-deployer
+WORKDIR /self-deployer
+RUN CGO_ENABLED=0 go build -o /output/deployer -v
+
+#Deploy Stage
+FROM alpine:latest
+
+LABEL maintainer="LitmusChaos"
+
+#Install kubectl
+
+ARG KUBE_VERSION="1.18.5"
+
+RUN apk add --update ca-certificates && \
+    apk add --update -t deps curl && \
+    curl -L https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl && \
```

Put an argument for the architecture also
AGR ARCH="amd64"

```suggestion
    curl -L https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/ARCH/kubectl -o /usr/local/bin/kubectl && \
```